### DISCUSSION TITLE
Add radius variable on sound command

### DISCUSSION THREAD

**Discussion by EagL3-GIT** [Zrips/CMI#7366] [Feature Requests]
Hi, I think there should be ability to play sound for players in radius of blocks.
[sound [sound] (-p:[pitch]) (-v:[volume]) (playerName/-all) (world) (x) (y) (z) (-s)](https://www.zrips.net/cmi/commands/]

**Comment by CloudeLecaw** [comment]
yep if its a location or at a play a radius would be cool.
may a 0 in case its just the player or just the locations else radius in blocks around target
might something like a -r:radius Option would be nice
